src/mmw/apps/bigcz/views.py

```diff
@@ -3,19 +3,43 @@
 from __future__ import unicode_literals
 from __future__ import division
 
+from django.contrib.gis.geos import GEOSGeometry
+from django.conf import settings
 from rest_framework import decorators
 from rest_framework.exceptions import ValidationError, ParseError
 from rest_framework.permissions import AllowAny
 from rest_framework.response import Response
 
 from apps.bigcz.clients import CATALOGS
 from apps.bigcz.serializers import ResourceListSerializer
-from apps.bigcz.utils import parse_date
+from apps.bigcz.utils import parse_date, get_bounds, \
```

Comment:
[Elsewhere](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/apps/modeling/mapshed/tasks.py#L12) in [the code](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/apps/geoprocessing_api/tasks.py#L19), multiline imports have been done using parentheses. Consider switching to those for consistency.

Reply:
Guess it depends where [you look](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/apps/bigcz/serializers.py#L8).  I was trying to keep it consistent within the django app, and I actually prefer this syntax over the parentheses since it allow you to start the next line indented one tab instead of at the opening parens, which can lead to a long imports section.

I'll update the two imports formatting.

Reply:
One import per line does make it easier to read diffs, but modern diffs highlight changed text in the same line so I think this'll be fine. 👍